README.md

```diff
@@ -636,6 +638,21 @@ their dependencies as well:
 - Add `:postgrex` for `AyeSQL.Runner.Postgrex`.
 - Add `:ecto_sql` and `:postgrex` for running queries using `Ecto` in a
   `PostgreSQL` database.
+- Add `:ecto_sql` and `:myxql` for running queries using `Ecto` in a
+  `MySql` database.
+
+### Using a particular adapter for an Ecto repo
```

Comment:
Maybe this could be summarized somewhere else, as this is more related to Ecto than AyeSQL.